settingtypes.txt

```diff
@@ -50,3 +50,6 @@ enable_stairs_replace_abm (Replace old stairs) bool false
 #    If enabled, use the engine's spawn search which does not check for a
 #    suitable starting biome.
 engine_spawn (Use engine spawn search) bool false
+
+#    Inform player of condition and location of new bones.
```

Comment:
Needs moving to just after 'share_bones_time' above.

Reply:
good catch